lib/plugins/input/command.js

```diff
@@ -1,6 +1,5 @@
 'use strict'
 var split = require('split2')
-var momenttz = require('moment-timezone')
```

Comment:
I assume momenttz was not used (but it will be soon, when we format dates in the command plugin). Ok for now.